libcodechecker/libhandlers/analyze.py

```diff
@@ -289,11 +302,18 @@ def main(args):
         with open(args.skipfile, 'r') as skipfile:
             metadata['skip_data'] = [l.strip() for l in skipfile.readlines()]
 
+    # Update metadata dictionary with old values.
+    metadata_file = os.path.join(args.output_path, 'metadata.json')
```

Comment:
What is the current state of this metadata files? The requirement of metadata makes us incompatible with scan-build. Do we need it? Should we make it optional?